server/app/views/applicant/ApplicantLayout.java

```diff
@@ -573,7 +574,10 @@ private static boolean onTiDashboardPage(Http.Request request) {
     } catch (URISyntaxException e) {
       logger.error("Could not get the path for uri {}", request.uri());
     }
+    if (currentPath == null) {
+      return false;
+    }
 
-    return currentPath != null && currentPath.equals(tiDashboardPath);
+    return currentPath.contains(tiDashboardPath) || currentPath.contains("/admin/tiGroups");
```

Comment:
suggestion (non-blocking): It feels a bit weird that one part of this statement checks a constructed URI value but the other part of this statement checks a hardcoded string. Could we have them either both check a constructed URI or both check a string?

Reply:
I agree.  I've refactored to get both paths from the routes.  I didn't like the hard-coded string since that would break if, for some reason, we change the path to the edit client or add client pages.  I also didn't see the need to convert the TI dashboard route to a URI since we can be sure of the format given that that path is coming from our Play routes.

Reply:
True! Good call removing the URI portion.